BitPlanner/Scenes/CraftPage.cs

```diff
@@ -143,6 +144,52 @@ private void CopyTreesAsText()
         DisplayServer.ClipboardSet(result.ToString().Trim());
     }
 
+    public void CopyTreesAsCSV()
+	{
+		var result = new StringBuilder();
+		var separator = new string('-', 64);
+
+		// Ny header enligt rekommendation
+		result.AppendLine("Item,Minimum Quantity,Maximum Quantity,In Stock");
+
+		foreach (var tab in _recipeTabs.GetChildren().Cast<RecipeTab>())
+		{
+			var root = tab.GetTreeRoot();
+			TraverseAndAppendCSV(root, result, 0); // Skicka med depth = 0
+			result.AppendLine(separator); // Separera träd
+		}
+
+		DisplayServer.ClipboardSet(result.ToString().Trim());
+	}
+
+	private void TraverseAndAppendCSV(TreeItem item, StringBuilder csv, int depth)
+	{
+		var id = item.GetMetadata(0).AsUInt64();
+		var craftingItem = _data.CraftingItems[id];
+
+		var quantity = item.GetMetadata(2).AsGodotArray();
+		var minQuantity = quantity[0].AsUInt32();
+		var maxQuantity = quantity[1].AsUInt32();
+
+		// Namn + Tier
+		var itemName = craftingItem.Tier > -1
+			? $"{craftingItem.Name} (T{craftingItem.Tier})"
+			: craftingItem.Name;
```

Comment:
```suggestion
		var itemName = item.GetText(0);
```

In case I will change the format of name I won't have to edit it here as well.